### PR TITLE
Fpg 60 player interaction detection

### PR DESCRIPTION
**Pull request by mana998**
Taking interactions from database based on logged in player or taking default ones.
When players are close it displays the message else it displays nothing.
Added new env variable to enable connection to the database from the server.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -72,11 +78,25 @@ io.on('connection', (socket) => {
...
+    let result = await response.json();
+    if (result.playerId) {
+      id = result.playerId;
+    }
+    //load player interactions
+    let url = `${process.env.URL}api/interaction`;
+    if (id) url += `?player_id=${id}`;
+    response = await fetch(url);
+    result = await response.json();
+    games[roomName].interactions = result;
```
Comment (dprzygocka) [inline app.js:98]: you just added the interactions to the game objects?

File: app.js [hunk 2]
```diff
@@ -72,11 +78,25 @@ io.on('connection', (socket) => {
     // import map class
     // eslint-disable-next-line global-require
     const { GameMap } = require('./public/models/GameMap');
-    const mapFile = `./private/assets/maps/map${Utils.getRandomNumber(1, 6)}.json`;
+    //const mapFile = `./private/assets/maps/map${Utils.getRandomNumber(1, 6)}.json`;
+    const mapFile = `./private/assets/maps/map0.json`;
```
Comment (dprzygocka) [inline app.js]: also this should be commented and the line before uncommented

File: app.js (hunk 1 above)
Comment (mana998) [reply app.js:98]: yes so it just gets loaded once when the game starts and then players choose random values from there so we don't need to query database all the time

File: app.js (hunk 2 above)
Comment (mana998) [reply app.js]: damn again... sorry didnt notice :D

Comment (mana998) [reply app.js]: fixed it

**Pull request merged**